playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -176,6 +177,10 @@ func prepareSbtFiles(lc *fs_tool.LifeCycle, pipelineFolder string, workingDir st
 	if err != nil {
 		return nil, err
 	}
+	_, err = exec.Command("cp", filepath.Join(workingDir, scioCommonConstants), absFileFolderPath).Output()
```

Comment:
Shouldn't "cp" be a constant like `cpCmd`?

Reply:
No, but we have the `shCmd` constant, so it will be consistent to move "cp" as well.